css/main.css

```diff
@@ -0,0 +1,128 @@
+@import "./reset.css";
+.calendar {
+  font-family: Arial, Helvetica, sans-serif;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  max-width: 706px;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.calendar__day {
+  width: 100px;
+  height: 100px;
+  background-color: #eee;
+  border: 1px solid black;
+  position: relative;
+  cursor: pointer;
+  transition-property: transform background-color;
+  transition: 500ms;
+}
+.calendar__day::before {
+  font-family: inherit;
+  font-size: 30px;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+.calendar__day:nth-child(1)::before {
+  content: "1";
+}
+.calendar__day:nth-child(2)::before {
+  content: "2";
+}
+.calendar__day:nth-child(3)::before {
+  content: "3";
+}
+.calendar__day:nth-child(4)::before {
+  content: "4";
+}
+.calendar__day:nth-child(5)::before {
+  content: "5";
+}
+.calendar__day:nth-child(6)::before {
+  content: "6";
+}
+.calendar__day:nth-child(7)::before {
+  content: "7";
+}
+.calendar__day:nth-child(8)::before {
+  content: "8";
+}
+.calendar__day:nth-child(9)::before {
+  content: "9";
+}
+.calendar__day:nth-child(10)::before {
+  content: "10";
+}
+.calendar__day:nth-child(11)::before {
+  content: "11";
+}
+.calendar__day:nth-child(12)::before {
+  content: "12";
+}
+.calendar__day:nth-child(13)::before {
+  content: "13";
+}
+.calendar__day:nth-child(14)::before {
+  content: "14";
+}
+.calendar__day:nth-child(15)::before {
+  content: "15";
+}
+.calendar__day:nth-child(16)::before {
+  content: "16";
+}
+.calendar__day:nth-child(17)::before {
+  content: "17";
+}
+.calendar__day:nth-child(18)::before {
+  content: "18";
+}
+.calendar__day:nth-child(19)::before {
+  content: "19";
+}
+.calendar__day:nth-child(20)::before {
+  content: "20";
+}
+.calendar__day:nth-child(21)::before {
+  content: "21";
+}
+.calendar__day:nth-child(22)::before {
+  content: "22";
+}
+.calendar__day:nth-child(23)::before {
+  content: "23";
+}
+.calendar__day:nth-child(24)::before {
+  content: "24";
+}
+.calendar__day:nth-child(25)::before {
+  content: "25";
+}
+.calendar__day:nth-child(26)::before {
+  content: "26";
+}
+.calendar__day:nth-child(27)::before {
+  content: "27";
+}
+.calendar__day:nth-child(28)::before {
+  content: "28";
+}
+.calendar__day:nth-child(29)::before {
+  content: "29";
+}
+.calendar__day:nth-child(30)::before {
+  content: "30";
+}
+.calendar__day:nth-child(31)::before {
+  content: "31";
+}
+.calendar__day:hover {
+  transform: translateY(-20px);
+  background-color: #ffbfcb;
+}
```

Comment:
don't forget to add empty line in the end of the file

Reply:
![image](https://user-images.githubusercontent.com/118932043/206293363-9971dd59-9d69-45ed-85b8-9b96e998729e.png)
???